### PR TITLE
[Excel] (PivotTable) Add PivotTable data source preview sample

### PR DESCRIPTION
**Pull request by alison-mk**

### CODE REVIEW

File: samples/excel/85-preview-apis/pivottable-source-data.yaml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to get information about the data source of a PivotTable. It returns the type and string representation of the data source.</p>
+          <p><em>Note: This sample works in Excel on Windows and Excel on the web. It doesn't work in Excel on Mac; this is a known issue.</em></p>
+        </section>
+
+        <section class="setup ms-font-m">
+          <h3>Set up</h3>
+          <p> This creates a data sheet and a PivotTable.</p>
```
Comment (AlexJerabek) [inline samples/excel/85-preview-apis/pivottable-source-data.yaml:101]: Not sure this label adds much. The term "data sheet" might not convey the relationship between the two worksheets and a more verbose description would be excessive.

Comment (alison-mk) [reply samples/excel/85-preview-apis/pivottable-source-data.yaml:100]: Good points. I'll remove it.

File: samples/excel/85-preview-apis/pivottable-source-data.yaml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to get information about the data source of a PivotTable. It returns the type and string representation of the data source.</p>
```
[the remark this answers is not in the document]
Comment (alison-mk) [reply samples/excel/85-preview-apis/pivottable-source-data.yaml:94]: I'm incorporating all your other suggestions, but I want to keep this sentence in the description. I think the explicit detail here help convey the purpose of the data source APIs; why one would want information about the PivotTable data source and what type of information can be returned.

**Review by alison-mk — COMMENTED** [review]
Thanks for the review!

**Pull request merged**